components/Vote/Homepage/VoteTabs.jsx

```diff
@@ -86,6 +90,41 @@ function VoteTabs() {
   const { t } = useTranslation('vote')
   const [value, setValue] = React.useState(0)
 
+  const today = dayjs().toISOString()
+
+  let openVotes = []
+  let upcomingVotes = []
+  let pastVotes = []
+
+  let checkDate = (todayDate, startDate, endDate, vote) => {
+    if (dayjs(todayDate).isBetween(startDate, endDate)) {
+      openVotes.push(vote)
+    } else if (dayjs(todayDate).isBefore(dayjs(startDate))) {
+      upcomingVotes.push(vote)
+    } else {
+      pastVotes.push(vote)
+    }
+  }
+
+  votesArray.map((vote) => {
```

Comment:
You should change this to `votesArray.forEach()` generally, you should only use `.map()` if you plan to do something with the returned array